lib/jxl/modular/encoding/encoding.cc

```diff
@@ -210,19 +210,19 @@ Status DecodeModularChannelMAANS(BitReader *br, ANSSymbolReader *reader,
             (y ? channel.Row(y - 1) - 1 : r - 1);
         pixel_type_w guess = (y ? rtop[0] : 0);
         if (run == 0) {
-          reader->ReadHybridUintClusteredHuffRleOnly(ctx_id, br, &v, &run);
+          reader->ReadHybridUintClusteredHuffRleOnly(br, &v, &run);
           sv = UnpackSigned(v);
         } else {
           run--;
         }
         r[0] = sv + guess;
-        for (size_t x = 1; x < channel.w; x++) {
+        for (size_t x = 1; x != channel.w; x++) {
```

Comment:
Why this change?

Reply:
Ah, no particular reason, I tried (unsuccessfully) to write that loop in some other way and that's a leftover from that attempt.